api/footnotes.asciidoc

```diff
@@ -162,3 +162,7 @@ Only once the ID has been allotted may it be exposed to OpenCL by proposing a me
 The merge must define a new enumerant by adding an `<enum>` tag to the {cl_khronos_vendor_id_TYPE} `<enums>` tag, with the `<value>` attribute set as the acquired Khronos vendor ID. \
 The `<name>` attribute must identify the vendor/adopter, and be of the form `CL_KHRONOS_VENDOR_ID_<vendor>`. \
 ]
+
+:fn-setkernelarg-prefer-unset-on-error: pass:n[ \
```

Comment:
The top of this file says

```c
// Please keep footnotes in alphabetical order!
```

But this can be moved afterwards, I don't want to block merging.

Reply:
I'll fix this.  If there are no other issues I think we should merge this now.

Reply:
Sorry I missed this.